app/src/ui/dialog/header.tsx

```diff
@@ -58,21 +58,14 @@ export class DialogHeader extends React.Component<IDialogHeaderProps, {}> {
       return null
     }
 
-    // We're intentionally using <a> here instead of <button> because
-    // we can't prevent chromium from giving it focus when the the dialog
-    // appears. Setting tabindex to -1 doesn't work. This might be a bug,
-    // I don't know and we may want to revisit it at some point but for
-    // now an anchor will have to do.
     return (
-      // eslint-disable-next-line jsx-a11y/anchor-is-valid, jsx-a11y/click-events-have-key-events, jsx-a11y/interactive-supports-focus
-      <a
+      <button
```

Comment:
I chose not to use our `<Button>` component here since this is clearly custom styled and I didn't want to introduce anything from our `<Button>` component.